dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3-slim
+
+# For healthcheck
+RUN apt-get update && apt-get install bash pip python git curl -y
```

Comment:
You may want to clear out the apt cache after installing these packages so the image size stays small(ish).

There's a good example here:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments